packages/components/src/Gallery/GalleryTypes.ts

```diff
@@ -35,6 +35,11 @@ export interface GalleryProps {
 
 export interface File
   extends Pick<FileUpload, "key" | "name" | "type" | "size" | "progress"> {
+  /**
+   * The thumbnail url of the file.
+   */
+  readonly thumbnailSrc?: string;
```

Comment:
now that i think about it, this should probs come from format file

Reply:
oh, nvm, it should be here as the format file isn't really gonna open the image and source there is the thumbnail size